servers/rendering/rendering_server_default.h

```diff
@@ -1049,6 +1049,10 @@ class RenderingServerDefault : public RenderingServer {
 	virtual void init() override;
 	virtual void finish() override;
 
+	virtual bool is_on_render_thread() override {
```

Comment:
Does this need to be a virtual function? It will never be overriden will it?

Reply:
Yes, because everything is exposed as template functions on `RenderingServer`, so if you ever implement a different rendering server, it will needs to implement its own threading logic.